src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -44,9 +46,12 @@ public class OrganizationStorageService extends BaseService implements Organizat
   @Override
   public CompletableFuture<Organization> createOrganization(Organization organization, Context context,
       Map<String, String> headers) {
+    // Should I add the organization object to log
+    logger.debug("createOrganization:: Trying to create organization with name: {}", organization.getName());
     HttpClientInterface client = getHttpClient(headers);
```

Comment:
Should I add the organization object to the log? Organization to the string may not be implemented. I am using only its name

Reply:
Basically organization id will be the most useful in logs, additionally you can provide organization name